applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -2313,3 +2294,25 @@ is_owner(Call, OwnerId) ->
         OwnerId -> 'true';
         _Else -> 'false'
     end.
+
+-spec send_mwi_update(mailbox(), whapps_call:call()) -> 'ok'.
+send_mwi_update(#mailbox{owner_id=OwnerId, mailbox_number=BoxNumber}=Box, Call) ->
+    AccountDb = whapps_call:account_db(Call),
+    _ = wh_util:spawn('cf_util', 'unsolicited_owner_mwi_update', [AccountDb, OwnerId]),
+    Messages = get_messages(Box, Call),
+    New = count_messages(Messages, ?FOLDER_NEW),
```

Comment:
I wonder if the `?FOLDER_*` macros should go somewhere else so `cf_util:vm_count/1` can use them instead of raw binaries? Maybe a `kzd_voicemail_box.hrl` in `core/kazoo_documents/include/` and add the appropriate `-include_lib()` in the modules?